common/workunit/workunit.cpp

```diff
@@ -8458,7 +8458,9 @@ void CLocalWorkUnit::copyWorkUnit(IConstWorkUnit *cached, bool copyStats, bool a
 bool CLocalWorkUnit::hasDebugValue(const char *propname) const
 {
     StringBuffer lower;
-    lower.append(propname).toLowerCase();
+    lower.append(propname);
```

Comment:
@jakesmith can you think of any problems this would cause?   I haven't found any.

Reply:
No, but I might be tempted to put these 3 common lines in a helper method/function, with a comment saying why it is differentiating.

Reply:
No, but I might be tempted to put these 3 common lines in a helper method/function, with a comment saying why it is differentiating.